### PR TITLE
yapf on deepchem/trans

### PR DESCRIPTION
**Pull request by maithili232**
# Pull Request Template

## Description

Fix # 3159 

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [ ] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [ ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ ] Run `python -m doctest <modified file>` and check no errors
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/trans/transformers.py [hunk 1]
```diff
@@ -20,1966 +20,1979 @@
...
+    References
+    ---------
+
+    .. [1] Polishchuk, P., et al. J. Chem. Inf. Model. 2016, 56, 8, 1455–1469
+    """
+
+    def __init__(self, dataset: Dataset):
+        """Initializes flattening transformation.
 
     Parameters
```
Comment (advikavs) [inline deepchem/trans/transformers.py:1063]: Lines 1063-67 need to be formatted.

**Review by advikavs — APPROVED** [review]
LGTM

**Review by gusty1g — APPROVED** [review]

**Review by gusty1g — CHANGES_REQUESTED** [review]
fix errors in mypy

**Comment by gusty1g** [comment]
This is a duplicate of #3170 and it can safely be closed.

**Pull request closed without merging**